src/app/components/LiveLabel/index.tsx

```diff
@@ -47,12 +49,19 @@ const LiveLabel = ({
     screenReaderText += ', ';
   }
 
+  const circle = [
+    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+  ];
+
   return (
     // The id below is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
     // eslint-disable-next-line jsx-a11y/aria-role
     <span id={id} role="text">
+      <span css={circle}>
+        <span css={styles.livelabelPulse} />
```

Comment:
Is this a self closing span element? Should it be <span css...></span>?

Reply:
The linting rules moan about empty components needing to be self-closing 😞

Reply:
This should be ok, React will render self-closing tags as `<span></span>` if given `<span />`.